From 6fe59e83aeee16ef1d5d565a7d9f87ffb8366b33 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@st.com>
Date: Wed, 1 Sep 2021 17:19:20 +0200
Subject: [PATCH 384/529] ARM: configs: disable Ethernet MICROCHIP Phy on
 fragment-01-multiv7_cleanup

When driver ethernet is unbind/bind, there is an error:
    [  688.292133] stm32-dwmac 5800a000.eth1: Failed to reset the dma
    [  688.296678] stm32-dwmac 5800a000.eth1 eth0: stmmac_hw_setup: DMA engine initialization failed
    [  688.309750] stm32-dwmac 5800a000.eth1 eth0: stmmac_open: Hw setup failed
This is link to config init of LAN88xx microchip driver
Until final fix found, disable this driver to use GENERIC Phy driver by default

Signed-off-by: Christophe Roullier <christophe.roullier@st.com>
Change-Id: I809f1ff30c579c580274a8b1a7ae89366133094e
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/218123
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/234736
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/284378
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 arch/arm/configs/fragment-01-multiv7_cleanup.config | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm/configs/fragment-01-multiv7_cleanup.config
+++ b/arch/arm/configs/fragment-01-multiv7_cleanup.config
@@ -64,6 +64,7 @@
 # CONFIG_KEXEC is not set
 # CONFIG_LOCKUP_DETECTOR is not set
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
+# CONFIG_MICROCHIP_PHY is not set
 # CONFIG_MTD_SPI_NOR_USE_4K_SECTORS is not set
 # CONFIG_PCI is not set
 # CONFIG_PCI_DOMAINS_GENERIC is not set
@@ -92,5 +93,6 @@
 # CONFIG_STAGING is not set
 # CONFIG_SUNXI_SRAM is not set
 # CONFIG_USB_DWC3 is not set
+# CONFIG_USB_LAN78XX is not set
 # CONFIG_USB_MUSB_HDRC is not set
 # CONFIG_VIRTIO_CONSOLE is not set
